azure-pipelines/build.yml

```diff
@@ -71,7 +76,7 @@ extends:
             publishVstsFeed: 97a41293-2972-4f48-8c0e-05493ae82010
           - output: nuget
             displayName: 'NuGet push to NuGet.org'
-            condition: and(and(succeeded(), eq(variables['Build.OfficialRelease'], 'true')), eq(variables['TimestampPackage'], 'false'))
+            condition: and(succeeded(), eq(variables['Build.OfficialRelease'], 'true'))
```

Comment:
Does `variables['Build.OfficialRelease']` need to be replaced now too?

Reply:
No, because we define it the variables section:

```yml
- name: Build.OfficialRelease
   value: ${{ parameters.OfficialRelease }}
```

Ideally, we'd find a way to avoid needing a Build.OfficialRelease variable at all and only use the parameter, but I was unable to figure out the syntax for that. Everything I tried caused a syntax error when running so I tried setting a variable to the value and that worked, so I went with it :-)